dictdiffer/utils.py

```diff
@@ -270,6 +270,17 @@ def are_different(first, second, tolerance):
         return not (first_is_nan and second_is_nan)
     elif isinstance(first, num_types) and isinstance(second, num_types):
         # two numerical values are compared with tolerance
-        return abs(first-second) > tolerance * max(abs(first), abs(second))
+        if tolerance is not None:
+            rel_diff = (
+                abs(first-second) > tolerance * max(abs(first), abs(second))
+            )
+        else:
+            rel_diff = True
+        # two numerical values are compared with absolute_tolerance
+        if absolute_tolerance is not None:
+            abs_diff = (abs(first-second) > absolute_tolerance)
+        else:
+            abs_diff = True
+        return rel_diff and abs_diff
```

Comment:
we could add a third case here where there is no tolerance (or absolute_tolerance=0 which would require the above comparison to become >=) so that you can allow only exact matches.

Reply:
Ignore this as I missed line 262.

Reply:
If I understand your point correctly, this case is already covered (thanks to `utils.py:262`), as shown by the test line 150.